package.json

```diff
@@ -50,5 +54,18 @@
     "transformIgnorePatterns": [
       "/node_modules/(?!antd|@ant-design|rc-.+?|@babel/runtime).+(js|jsx)$"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
```

Comment:
@MigueMartelo  No es buena practica usar el símbolo ^ en prod ya que eso puede hacer que se descarguen versiones actuales de una lib para que el sistema no esta listo, incluso si se tiene el package lock es buena practica tener la versión exacta.

Si puedes de una vez hacer pin a las demás dependencias estaría genial!

Algunas referencias:
https://docs.renovatebot.com/dependency-pinning/
https://the-guild.dev/blog/how-should-you-pin-dependencies-and-why

Reply:
Done! ✅